tools/pipelines-tasks/AVDAppAttachPublish/main.ts

```diff
@@ -75,44 +76,48 @@ async function run(): Promise<void> {
 		};
 
 		const jsonString = JSON.stringify(appattachConfig);
-		tl.debug(jsonString);
+		appattachConfig['accessToken'] = "***";
+		appattachConfig['azureStorageKey'] = "***";
+		tl.debug(JSON.stringify(appattachConfig));
```

Comment:
Nit: Maybe add comment that this is to not show secrets in logging. Or maybe you could log each property on its own instead of the whole JSON, and just skip these two

Reply:
Done